testapps/svelte/.gitignore

```diff
@@ -3,7 +3,7 @@ node_modules
 /build
 /.svelte-kit
 /package
+.env
+.env.*
+!.env.example
```

Comment:
We really want to ignore just the .local environments...

Reply:
fixed